rasa/core/migrate.py

```diff
@@ -283,5 +283,7 @@ def migrate_domain_format(
     _write_final_domain(domain_file, new_forms, new_slots, out_file)
 
     rasa.shared.utils.cli.print_success(
-        f"Your domain was successfully migrated! It's location is {out_file}."
+        f"Your domain file '{str(domain_file)}' was successfully migrated! "
+        f"The migrated version is now '{str(out_file)}'. "
+        f"The original domain file is backed-up as 'original_domain.yml'."
```

Comment:
The back-up location shouldn't be hardcoded, because if a directory path was provided as domain, then the back-up location will be the `original_domain` directory, otherwise if a `domain.yml` file was provided as domain, then the back-up location will be `original_domain.yml`. 

Probably best to align the two variables in this function in the above if statement to have same name (`backup_dir` and `backup`) and then provide this as something along the lines of  `The original domain is backed-up in '{str(backup_path)}'`

Reply:
I needed to define `backup_path` in more than one if statement. Does this change work better?